### ISSUE TITLE
Organisation of 3.4 "Chartered Groups: Working Groups and Interest Groups"

### ISSUE THREAD

**Issue by mnot** [w3c/w3process#554]
I found it a bit odd that 3.3 had separate subsections for the TAG and AB, and this section did not; instead, it intersperses requirements for the two types of chartered groups in alternating sections.

An easy reorganisation would be to move 3.4.3.[1,3,5] into a subsection of 3.4, 'Working Groups' and 3.4.3.[2,4,6] into a subsection 'Interest Groups'.

(Editorial, obviously)

**Comment by frivoal** [comment]
Yeah. I was a bit unsure when we worked on this section. it's a bit of a 2D system: (WG,IG) x (member rep, IE, team rep). I thought about organizing it the way you suggest, but the way the actual prose is written flows better in the current order, because all the IG sections come immediately after the WG section for the same type of participant, and they say something like "same as the WG, except [differences]". That would still work if they were grouped differently, but I thought it read better this way. That said, I think I could be convinced it's better the other way around, with an organization like what you suggested. No strong feelings either way.

**Comment by mnot** [comment]
From a reader standpoint, I think most will want to identify the relevant section by the type of group; that makes the document easier to scan. People don't often read this type of document in one serial sitting...

**Comment by TallTed** [comment]
> From a reader standpoint, I think most will want to identify the relevant section by the type of group; that makes the document easier to scan. People don't often read this type of document in one serial sitting...

True.  By this logic, however, the content which is the same should be reproduced in both sections, rather than omitted in favor of a briefer "this later section differs from that earlier section in that x, y, and z" -- because anyone looking to the type of group for the info they need is not going to like needing to read the section about the other type of group first.

**Comment by dwsinger** [comment]
We could look at re-organizing into a common considerations section, working groups, and interest groups, sure. The outline suggests that that might be a way to go.
